lib/routemap.c

```diff
@@ -1691,6 +1708,12 @@ static void route_map_clear_all_references(char *rmap_name)
 	}
 }
 
+static unsigned int route_map_dep_data_hash_make_key(void *p)
+{
+	struct route_map_dep_data *dep_data = p;
+	return (string_hash_make(dep_data->rname));
```

Comment:
Line between declerations and function, remove the paranthesis around the return statement